cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -124,37 +124,64 @@ func UpdateAllInstanceNoExternalIP(workflow *daisy.Workflow, noExternalIP bool)
 	})
 }
 
+// SetupUEFIUpdateHook setup pre-run hook for each create-images steps to
+// update "UEFI_COMPATIBLE" tag based on context
+func SetupUEFIUpdateHook(workflow *daisy.Workflow) {
+	workflow.IterateWorkflowSteps(func(s *daisy.Step) {
+		if s.CreateImages != nil {
```

Comment:
We should also check for instance name here. It's unlikely that another instance will output is-uefi-compatible in a workflow, but it's better to be certain the correct instance did output this value

Reply:
The instance name contains a random suffix.
Even we can get that suffix somehow, I don't feel it's really better: it assumes an instance name which is given by the workflow.
Current PR: assumes no other instances called translate_bootstrap script, which is also defined by workflow.
Both are based on an assumption towards the workflow.